dotcom-rendering/src/web/components/CalloutBlockComponent.stories.tsx

```diff
@@ -0,0 +1,93 @@
+import { ArticleDesign, ArticleDisplay, ArticlePillar } from '@guardian/libs';
+import fetchMock, { FetchMockStatic } from 'fetch-mock';
+import { calloutCampaign as calloutCampaignV2 } from '../../../fixtures/manual/calloutCampaignV2';
+import { CalloutBlockComponent } from './CalloutBlockComponent.importable';
+
+const mockFormat = {
+	display: ArticleDisplay.Standard,
+	design: ArticleDesign.Standard,
+	theme: ArticlePillar.Opinion,
+};
+
+const tomorrow = new Date().setDate(new Date().getDate() + 1) / 1000;
```

Comment:
This works but seems slightly long winded - if anyone knows an easier way to set a date roughly a few days in the future/past please let me know!